aldryn_redirects/models.py

```diff
@@ -43,3 +47,40 @@ def __str__(self):
         if not new_paths:
             new_paths = ugettext('None')
         return "{} ---> {}".format(self.old_path or 'None', new_paths)
+
+
+class StaticRedirect(models.Model):
+    sites = models.ManyToManyField('sites.Site', related_name='+')
+    inbound_route = models.CharField(
+        max_length=255,
+        db_index=True,
+        validators=[validate_inbound_route, ],
+        help_text=_('Redirect origin. Do not provide the domain. Always add a leading slash here.'),
+    )
+    outbound_route = models.CharField(
+        max_length=255,
+        validators=[validate_outbound_route, ],
+        help_text=_('Redirect destination. Domain is not required (defaults to inbound route domain).'),
+    )
+
+    objects = StaticRedirectManager.as_manager()
+
+    def get_outbound_url(self, request):
```

Comment:
please avoid methods which require a request object.
these can be quite difficult to debug and maintain.
I suggest `get_outbound_url(self, site, scheme)`